src/editor/objects/AudioSource.js

```diff
@@ -231,8 +231,12 @@ export default class AudioSource extends Object3D {
 
   async load(src) {
     await this.loadMedia(src);
-    this.audioSource = this.audioListener.context.createMediaElementSource(this.el);
```

Comment:
Can we add attributions to AudioNodes as well?

Reply:
This is actually unrelated to the attribution feature, it's related to what I mention in the PR notes: 

> It also fixes an issue with adding new source to audio/video nodes that threw an exception in Chrome an the updated element didn't work:
> 
> Cause:
>     Failed to execute 'createMediaElementSource' on 'AudioContext': HTMLMediaElement already connected previously 

But yep, I should have split that in another PR...